### PR TITLE
feat: add the mode command + read the groups_used from env

### PR DESCRIPTION
**Pull request by dtrckd**
I have added the GROUPS_USED variable to the environments.
 @bolinocroustibat

### CODE REVIEW

**Comment by bolinocroustibat** [comment]
Has it been added to GitHub Actions environment secrets? CD/CI is now using GitHub Actions secrets which are not yet updated automatically when commiting on `environments` repo.

**Comment by dtrckd** [comment]
> Has it been added to GitHub Actions environment secrets? CD/CI is now using GitHub Actions secrets which are not yet updated automatically when commiting on `environments` repo.

Yes.

**Review by bolinocroustibat — CHANGES_REQUESTED** [review]

File: app/commands.py [hunk 1]
```diff
@@ -170,6 +170,25 @@ async def albert_conversation(ep: EventParser, matrix_client: MatrixClient):
...
+    onEvent=RoomMessageText,
+    command="mode",
+    help=f"**{COMMAND_PREFIX}mode** MODE : Modifier le mode du modèle (c'est-à-dire le modéle de prompt utilisé).",
+)
+async def albert_mode(ep: EventParser, matrix_client: MatrixClient):
+    config = user_configs[ep.sender]
+    await matrix_client.room_typing(ep.room.room_id)
+    commands = ep.event.body.split()
+    if len(commands) <= 1:
+        reset_message = "La comande !mode necesite un argument. Se référer à !help."
```
[the remark this answers is not in the document]
Comment (dtrckd) [reply app/commands.py:184]: But if the group is mentioned in GROUPS_USED it won't be hidden, no ?
I have added "albert_debug" for the staging env.

[the remark this answers is not in the document]
Comment (bolinocroustibat) [reply app/commands.py]: Do all commands in the group "albert_debug" are not displayed in the help, right?

[the remark this answers is not in the document]
Comment (bolinocroustibat) [reply app/commands.py]: OK gotcha, it is, but we don't use "albert_debug" in any other place than staging. LGTM

**Review by bolinocroustibat — APPROVED** [review]
LGTM, thanks

**Pull request merged**